### ISSUE TITLE
remove RefSeq / Organism from top bar in favor for "currentOrganism": switch on clicking row

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#273]
From @monicacecilia   on #215.

* Current organism is for example "Honey Bee"
* I have looked at the available "Tracks" on the side panel
* Then I choose the "Ref Sequence" Tab.
* Then I change to a different organism, for example "Human"
* The "Ref Sequence" tab refreshes and shows the new available ref sequences, HOWEVER, when I click back to "Tracks", it still shows those of the previous organism. oh... wait.... and if I change to a third organism.... a mess.
* Then, I decided to refresh this page and the mysterious error cleared up. I have a feeling this has to do with some "lag" in the connection. ... I am now just as puzzled as you are @nathandunn .

**Comment by nathandunn** [comment]
I think that this is important to get right.

**Comment by monicacecilia** [comment]
Alas, so the issue I described is fixed, but now the opposite is true. 

* Current organism is for example "Yeast"
* I have looked at the available "Tracks" on the side panel
* Then I choose the "Ref Sequence" Tab.
* Then I change to a different organism, for example "Honey1"
* The "Ref Sequence" tab DOES NOT refresh and shows the reference sequences available for the previous organism. The "Tracks" tab shows updated lists. 
* When refreshing, the mistake goes away. 

:anguished:
